jerry-core/mem/mem-heap.cpp

```diff
@@ -188,7 +188,7 @@ typedef struct
 /**
  * Heap
  */
-mem_heap_t mem_heap;
+mem_heap_t mem_heap JERRY_HEAP_SECTION_ATTR;
```

Comment:
@tczene, thank you for the description.

Maybe, we could set the option not to `__attribute__((section(.jerry_mem)))`, but to `.jerry_mem`, so simplifying scripts and code.
In the way, we could provide `.bss` as default value for the macro.

What do you think about this?